.github/workflows/cpp.yml

```diff
@@ -56,17 +56,20 @@ jobs:
 
   build:
     runs-on: ubuntu-latest
-    
+    strategy:
+      matrix:
+        # TODO(mathetake): Add other runtimes.
+        wasm_runtime: [ "wasmtime" ]
```

Comment:
`runtime=XXX` should be self-explanatory in this context, adding `wasm` everywhere adds more noise than value.